elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -58,6 +66,9 @@
     private final MultipleFilterDialect filterDialect;
     private final Map<String, FilterExpression> expressionsByType;
 
+    @Getter private final Map<String, Long> checkStats;
+    @Getter private final Set<Triple<Class, Class, String>> expressionResultShortCircuit;
```

Comment:
Moving it(and the checkStats) to `ActivePermissionExecutor` should also require the `PermissionExecutor` to have the getter declaration. I'm not sure if it makes sense for `BypassPermissionExecutor` to have these getters.